.golangci.yml

```diff
@@ -281,4 +281,7 @@ issues:
       path: private/buf/cmd/buf/buf_test.go
       # The LsModules tests call chdir and cannot be parallelized.
       text: "LsModules"
-      
+    - linters:
+        - forbidigo
+      # We are using errgroup in a test for concurrent reads/writes to the cache.
+      path: private/bufpkg/bufmodule/bufmodulecache/bufmodulecache_test.go
```

Comment:
We might be able to refactor the test to use thread.Parallelize - I don't think we're relying on any specific behavior from errgroup there?